extra/aws/external-worker-aws-cf-template.yaml

```diff
@@ -401,8 +448,16 @@ Resources:
           - ''
           - - "#!/bin/bash -xe\n"
 
-            - "export STACK_NAME="
-            - Ref: AWS::StackId
+            - "export PROJECT="
+            - Ref: AWS::StackName
```

Comment:
Just to be sure, previous STACK_NAME contained the ID             - Ref: AWS::StackId	
and was used to send the CF signal.

I saw that you switched in aws cli to the stack name instead of the ID (are both are the same, can you confirm it is working ?)

Reply:
it's working yes
```
--stack-name (string)

The stack name or unique stack ID that includes the resource that you want to signal.
```